octobot_evaluators/evaluator/TA_evaluator.py

```diff
@@ -52,7 +47,40 @@ async def start(self, bot_id: str) -> bool:
             self.logger.error("Can't connect to OHLCV trading channel")
         return False
 
+    async def reset_evaluation(self, cryptocurrency, symbol, time_frame):
+        self.eval_note = START_PENDING_EVAL_NOTE
+        await self.evaluation_completed(cryptocurrency, symbol, time_frame, eval_time=0)
+
+    async def inner_ohlcv_callback(self, exchange: str, exchange_id: str,
```

Comment:
Concerning the naming of this method, I think it's a bit confusing for an evaluator developer. I would prefer to have `ohlcv_callback` in my evaluator and the default evaluator callback to be something like `evaluator_ohlcv_callback`. What do you think ?

Reply:
good point !